core/src/test/java/greencity/controller/EventControllerTest.java

```diff
@@ -269,4 +270,37 @@ void updateEvent_ShouldReturnBadRequest_WhenDtoInvalid() throws Exception {
         verifyNoInteractions(userService, eventService, modelMapper);
     }
 
+    @Test
+    @WithMockUser(username = "User", roles = "USER")
```

Comment:
@WithMockUser doesn't work without @WebMvcTest or SpringExtension.class. Use principal in perform method instead.